### DISCUSSION TITLE
New `rules_rust` maintainer!

### DISCUSSION THREAD

**Discussion by scentini** [bazelbuild/rules_rust#2421] [Announcements]
Hi all, please welcome our new `rules_rust` maintainer @daivinhtran; for the last 1.5 years @daivinhtran has been working on Bazel build rules for the Android platform; going forward they will help us evolve the ruleset and keep up with the increased rate of community contributions.

cc @UebelAndre @illicitonion @krasimirgg

**Comment by UebelAndre** [comment]
Welcome! So glad to have you and am very excited to see more activity building Rust apps for Android! 🤖 ❤️

**Comment by illicitonion** [comment]
Welcome! Thanks so much for all of your contributions so far, excited to work more together!
